TGC/events/Germany VS Hungary Events.txt

```diff
@@ -0,0 +1,672 @@
+##Treaty of Vienna## 
+
+country_event = {
+	id = 2000000000
+	title = "EVTNAME2000000000"
+	desc = "EVTDESC2000000000"
+	picture = "german_rhine"

```

Comment:
I'm just nitpicking now but I'm not sure about this picture for this chain...

Reply:
im planning to add better pics (mostly of maps) to the events as soon as the code itself is done